### DISCUSSION TITLE
Column type for list of references

### DISCUSSION THREAD

**Discussion by davhad** [AntlerVC/firetable#974] [Support / Q&A] [answered]
Using Flutterflow x Rowy and trying to match a collection property of type list<Reference>. What would you suggest to have as a column in Rowy? Trying to have the data easily readable and maybe editable within Rowy.

I tried connector table and json but they seem to be blocking the record view page when it's opened.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @davhad, there is a known bug where Rowy currently doesn't support array docs refs. We're working on a fix for it, thanks for bringing this up!
Also, we're looking into appropriately mapping FlutterFlow field types with Rowy field types. We're planning to create a separate section for **Rowy x FlutterFlow** users and the documentation for this will be updated soon! Stay tuned! 🚀
